samples/vue/src/components/GraphQL/GraphQL-SSRDemo.vue

```diff
@@ -6,30 +6,34 @@
 
```

Comment:
Can you revert back changes in this file? I believe, it was done by your formatter